tables/c_tbtopn.c

```diff
@@ -90,6 +91,8 @@ function value          o: table descriptor
                         return NULL;
                     }
                 }
+                else
```

Comment:
Please remove the changes to tables/c_tbtopn.c.  It is not an error for `hdunum` to be different from 1.